numpy/lib/stride_tricks.py

```diff
@@ -111,6 +112,196 @@ def as_strided(x, shape=None, strides=None, subok=False, writeable=True):
     return view
 
 
+def _sliding_window_view_dispatcher(x, window_shape, axis=None, *,
+                                    subok=None, writeable=None):
+    return (x,)
+
+
+@array_function_dispatch(_sliding_window_view_dispatcher, module='numpy')
+def sliding_window_view(x, window_shape, axis=None, *,
+                        subok=False, writeable=False):
+    """
+    Create a sliding window view into the array with the given window shape.
+
+    Creates a sliding window view of the N dimensional array with the given
+    window shape. Window slides across each dimension of the array and extract
+    a subsets of the array at any window position.
+
+    Parameters
+    ----------
+    x : array_like
+        Array to create the sliding window view from.
+
```

Comment:
Are spaced between variables common? I thought normally these should be packed with only indentation indicating a variable after a description.